tests/test_send_stats_to_performance_platform.py

```diff
@@ -0,0 +1,359 @@
+import json
```

Comment:
Yeah, don't do it on my behalf. I would have just done the tests as end-to-end tests using requests mock to make sure the whole thing works.